### PR TITLE
Get started page 

### LINKED ISSUE #29: Get started (install)
The getting started (install) page is having a major redesign. Implement the new design as per the Figma designs. If this ticket can/should be broken down please feel free to talk with me and we can do that.

## Acceptance criteria

* The new page matches the Figma designs
* Design sign off
* The new page is accessible
* KPIs must be as good as, or better, than existing KPIs

## Considerations

* Google Analytics must still work as before

### PR DESCRIPTION
**Pull request by MariamM110**
closes #29 

- Page matches figma design
- Responsive 
- Some code was changed in hero section

### CODE REVIEW

File: assets/js/bioconductor.js
Comment (SociableSteve) [inline assets/js/bioconductor.js:1]: Let's open a ticket to see if we can CSS this.

File: assets/style/get-started.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
+.install-hero {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-top: 0.5rem;
+}
+
+.get-started-h1 {
+  color: var(--misc-dark) !important;
```
Comment (SociableSteve) [inline assets/style/get-started.css:10]: Do we need `!important`? This is normally indicative of a problem.

**Review by SociableSteve — DISMISSED** [review]
A couple of comments but I'm not going to block this going.

File: assets/style/get-started.css (hunk 1 above)
Comment (MariamM110) [reply assets/style/get-started.css:10]: the hero h1 has the blue default colour so it wouldnt allow me to override it unless i added important

**Review by bp289 — DISMISSED** [review]

**Review by SociableSteve — APPROVED** [review]

**Pull request merged**